src/client/driver/index.js

```diff
@@ -10,6 +10,7 @@ import * as browser from '../browser';
 import executeActionCommand from './command-executors/execute-action-command';
 import executeWaitForElementCommand from './command-executors/execute-wait-for-element-command';
 import executeNavigateToCommand from './command-executors/execute-navigate-to-command';
+import executePendingBrowserWindowCommand from './command-executors/execute-pending-browser-window-command';
```

Comment:
I'm sorry, now I think we shouldn't have `_addPendingBrowserWindowTask `, because really we don't have pending browser window task. So this method should be renamed into `_addPendingBrowserWindowCommand'`